lib/temporal/workflow/context.rb

```diff
@@ -130,6 +130,7 @@ def execute_workflow(workflow_class, *input, **args)
           parent_close_policy: parent_close_policy,
           timeouts: execution_options.timeouts,
           headers: execution_options.headers,
+          cron_schedule: execution_options.cron_schedule,
```

Comment:
We should handle it similarly to `parent_close_policy` and not add it to `execution_options` since these were meant for overlapping properties between Activities and Workflows. We might consider subclassing these later to host all options, but it feels like a separate refactor